recipes/iverilog/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "10.1" %}
+{% set git_tag = "v10_1" %}
+
+package:
+  name: iverilog
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/steveicarus/iverilog.git
+  git_tag: {{ git_tag }}
```

Comment:
The `url` should then be `https://github.com/steveicarus/iverilog/archive/{{ git_tag }}.tar.gz`.